bigbluebutton-html5/imports/ui/components/nav-bar/talking-indicator/component.jsx

```diff
@@ -61,6 +68,10 @@ class TalkingIndicator extends PureComponent {
           className={cx(style)}
           onClick={() => this.handleMuteUser(voiceUserId)}
           label={callerName}
+          tooltipLabel={!muted && amIModerator
+            ? `${intl.formatMessage(intlMessages.muteLabel)} ${callerName}`
```

Comment:
Do we need to check all conditions here? If the condition was false wouldn't we be omitting the button?

Reply:
If it's false, the button is still visible it just doesn't get a tool-tip applied